configure

```diff
@@ -1439,19 +1488,22 @@ char *xgetcwd(void) {
 }
 int main(void) {
     char *wd1 = getcwd(NULL, 0);
-    if (!wd1) return 1;
+    if (!wd1) return 0;
     char *wd2 = xgetcwd();
-    if (!wd2 || strcmp(wd1, wd2) != 0) return 1;
+    if (!wd2 || strcmp(wd1, wd2) != 0) return 0;
     char *wd11 = realloc(wd1, strlen(wd1) + 10);
-    if (!wd11 || strcmp(wd11, wd2) != 0) return 1;
-    free(wd11); free(wd2); return 0;
+    if (!wd11 || strcmp(wd11, wd2) != 0) return 0;
+    free(wd11); free(wd2); return 1;
 }
 END
-trymake && tryexec
-checked
+    trymake && tryexec
+    checked
+else
+    checked "$YASH_GETCWD_NO_AUTO_MALLOC"
+fi
 if [ x"${checkresult}" = x"yes" ]
 then
-    defconfigh "GETCWD_AUTO_MALLOC"
+    defconfigh "GETCWD_NO_AUTO_MALLOC"
```

Comment:
Could you explain why you replaced `GETCWD_AUTO_MALLOC` with `GETCWD_NO_AUTO_MALLOC`?